tests/framework/installer/ceph_installer.go

```diff
@@ -494,12 +494,18 @@ func (h *CephInstaller) InstallRook(namespace, storeType string, usePVC bool, st
 	useCrashPruner := rookVersion == VersionMaster
 
 	// Create rook cluster
-	err = h.CreateRookCluster(namespace, onamespace, storeType, usePVC, storageClassName,
-		cephv1.MonSpec{Count: mon.Count, AllowMultiplePerNode: mon.AllowMultiplePerNode}, startWithAllNodes,
-		skipOSDCreation, useCrashPruner, h.CephVersion)
-	if err != nil {
-		logger.Errorf("Rook cluster %s not installed, error -> %v", namespace, err)
-		return false, err
+
+	maxTry := 3
+	for i := 0; i < maxTry; i++ {
+		err = h.CreateRookCluster(namespace, onamespace, storeType, usePVC, storageClassName,
+			cephv1.MonSpec{Count: mon.Count, AllowMultiplePerNode: mon.AllowMultiplePerNode}, startWithAllNodes,
+			skipOSDCreation, useCrashPruner, h.CephVersion)
+		if err == nil {
+			break
+		} else if i == maxTry-1 {
```

Comment:
nit: since there is a `break`, no need to put the next conditional in an `else`
```suggestion
		} 
	        if i == maxTry-1 {
```

Reply:
Done.